README.md

```diff
@@ -1,159 +1,9 @@
-
-* **withComponents (Higher Order Components)** – VulcanJS makes a lot of use of React’s [higher order components](https://reactjs.org/docs/higher-order-components.html). Often you need a component to have access to particular attributes (the currentUser object, or a function that can edit a given document). Vulcan has a series of components that wrap around other components. They have a names like `withCurrentUser`, `withEditMutation`, `withDocument`. Wrapping a component in a `withX` typically passes extra information in as a prop.
-
-* **Fragments** – GraphQL queries are made using fragments, which describe the fields from a given database object you want to fetch information on. There’s a common failure mode where someone forgets to update a fragment with new fields, and then the site breaks the next time a component attempts to use information from the new field.
-
-A Vulcan project is divided into packages. It comes with several base packages that provide core functionality (users, voting, routing, etc).
-
-The two packages you’ll most likely need to pay attention to are:
-
-* `packges/example-forum` – A sample forum that defines some key collections (posts, comments, and notifications, as well as “categories” [basically tags] which we aren’t currently using).
-
-  We avoid editing example-forum if we can, so we can more easily upgrade it when the main Vulcan branch updates. Instead, if a function in example-forum doesn’t suit our needs, we replace in our custom lesswrong package.
-
-  However, in some cases it’s impractical to avoid changing example-forum. When we do, we add a nearby comment:
-
-      // LESSWRONG – [text explaining the change]
-
-  So we can easily search for LESSWRONG and find all changes we’ve made.
-
-* `packages/lesswrong` – Our primary codebase.
-
-This includes new collections, components, and additional functions/attributes/extensions for objects in example-forum.
-
-### Development Tips
-
-#### Iteration
-* Prefer `_.range(n).forEach(i => my_function())` over `for (var i=0; i<n; i++)...`
-* If the body of a for loop performs a stateful action (i.e. modifies a variable outside the scope of the for body), use `forEach`. Else, use `map`.
-* Use underscore.js when possible.
-
-#### Style guide
-
-* [Syntax rules](https://github.com/Khan/style-guides/blob/master/style/javascript.md#syntax)
-* [Comments and documentation](https://github.com/Khan/style-guides/blob/master/style/javascript.md#comments-and-documentation)
-* [ES6 rules](https://github.com/Khan/style-guides/blob/master/style/javascript.md#es67-rules)
-
-### Debugging
+We have not prioritized writing a guide to the tech. You can see the instructions on the LessWrong repo, but any given sentence is more likely than not to be out of date and misleading. If you are at least moderately interested in contributing, message [JP](https://forum.effectivealtruism.org/users/jpaddison) on the the Forum and I might update this README.
```

Comment:
Minor: 'I might update this README' feels a little unfriendly, not sure if this is the intent, but it might discourage someone from trying to contribute. Maybe something like 'who can give you some pointers for getting started'. Obviously that might be overcommitting, so you could walk it back a bit depending on what you're comfortable with.